backend/clubs/views.py

```diff
@@ -4013,6 +4014,22 @@ def get_queryset(self):
         )
 
 
+class ExternalMemberListViewSet(viewsets.ModelViewSet):
```

Comment:
This should probably be a [ReadOnlyViewSet](https://www.django-rest-framework.org/api-guide/viewsets/#readonlymodelviewset), right?